app/views/helpers/fee_display_helper.rb

```diff
@@ -8,4 +8,11 @@ def self.grouped_by_display(fee)
 
     " • #{fee.grouped_by.values.compact.join(" • ")}"
   end
+
+  def self.should_display_subscription_fee?(invoice_subscription)
+    return false if invoice_subscription.blank?
+    return true if invoice_subscription.charge_amount_cents.zero?
```

Comment:
If I understand correct, this would display the subscription fee is the amount is zero 🤔 which seems to be the opposite of what we're trying to do.

```suggestion
    return false if invoice_subscription.charge_amount_cents.zero?
```

Reply:
@julienbourdeau  no, actually it is an exception to display the subscription if there is no other charge fee to display

Reply:
Oh yeah I see 👌